app/i18n/en/conf.php

```diff
@@ -142,7 +142,7 @@
 		'_' => 'Shortcuts',
 		'article_action' => 'Article actions',
 		'auto_share' => 'Share',
-		'auto_share_help' => 'If there is only one sharing mode, it is used. Else modes are accessible by their number.',
+		'auto_share_help' => 'If there is only one sharing mode, it is used. Other modes are accessible by their number.',
```

Comment:
I think that "else" was used as in "otherwise" here - not "other", which would make more sense.